Data/HashMap/Internal.hs

```diff
@@ -914,10 +924,11 @@ setAtPosition i k x ary = A.update ary i (L k x)
 
 
 -- | In-place update version of insert
-unsafeInsert :: (Eq k, Hashable k) => k -> v -> HashMap k v -> HashMap k v
+unsafeInsert :: forall k v. (Eq k, Hashable k) => k -> v -> HashMap k v -> HashMap k v
```

Comment:
Why do we need this? Do you implement the "let should not be generalized" paper unconditionally?

Reply:
Yes, MonoLocalBinds is always on. 
It's usually not a problem, but for runST its annoying.